_includes/apply_form.html

```diff
@@ -10,18 +10,18 @@ <h4 class="modal-title" id="myModalLabel4">{{ include.title }}</h4>
                     <div id="{{ include.target }}-fields">
                         <div class="row">
                             <div class="col-md-6 margin-bottom-20">
-                                <label>First Name</label><input name="{{ include.target }}-firstname" class="form-control" type="text" />
+                                <label>First Name (*)</label><input name="{{ include.target }}-firstname" class="form-control" type="text" />
```

Comment:
You can add required option to the field (e.g. `<input type="text" name="..." required>`)